content/docs/workflow-examples/tekton/tekton.md

```diff
@@ -0,0 +1,7 @@
+---
+title: serverless-workflow-examples tekton
+date: "2024-03-17"
+---
+
+{{< remoteMD "https://github.com/ydayagi/serverless-workflow-examples/blob/flpath938/tekton/README.md?raw=true" >}}
+
```

Comment:
Pls don't merge until the referenced PR is merged and this link points to the parodos-dev repository.

Reply:
other PR is merged

Reply:
fixed the link